BetterStats/package.ps1

```diff
@@ -0,0 +1,71 @@
+param ($vertype = 'patch')
```

Comment:
A build script I use in my own projects. Seems to make packaging releases a little less error prone.